app/src/main-process/main.ts

```diff
@@ -262,9 +265,10 @@ function createWindow() {
       rendererReadyTime: window.rendererReadyTime!,
     })
 
-    if (launchURLAction) {
-      window.sendURLAction(launchURLAction)
-      launchURLAction = null
+    const fns = onDidLoadFns!
```

Comment:
This makes me nervous, it seems like this could indeed be null the second time we create a window?